src/components/search/Search.jsx

```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
     setShowAlert(props.error)
   }, [props.error])
 
+  const options = searchConfig.map(config => (<option key={config.authority} value={config.authority}>{config.label}</option>))
+
   return (
     <div id="search">
       <Header triggerEditorMenu={props.triggerHandleOffsetMenu} />
       {alert}
-      <Grid>
-        <Row className="text-center">
-          <Form>
-            <FormGroup controlId="formGroupSearch" >
-              <FormControl type="text"
-                           placeholder="Search"
-                           aria-label="Search"
-                           onChange={ event => setQueryString(event.target.value) }
-                           onKeyPress={ event => handleKeyPress(event) } />
-              <span className="help-block">Use a * to wildcard your search.</span>
-            </FormGroup>
-          </Form>
-        </Row>
-        <SearchResults {...props} key="search-results" />
-        <SearchResultsPaging {...props} pageSize="1"/>
-        <SearchResultsMessage />
-      </Grid>
+      <div className="row">
+        <form className="form-inline" onSubmit={handleSubmit}>
+          <div className="form-group col-md-4" style={{ paddingRight: '5px' }}>
+            <label htmlFor="searchType" style={{ marginRight: '5px' }}>Search</label>
+            <select className="form-control" id="searchType"
+                    value={authority} style={{ paddingRight: '0px', width: '85%' }}
+                    onChange={ event => setAuthority(event.target.value) }
+                    onBlur={ event => setAuthority(event.target.value) }>
+              <option value="sinopia">Sinopia</option>
+              {options}
+            </select>
+          </div>
+          <div className="form-group col-md-6" style={{ paddingLeft: '5px' }}>
+            <div className="input-group" style={{ width: '100%' }}>
+              <label className="sr-only" htmlFor="searchInput">Query</label>
+              <input id="searchInput" type="text" style={{ width: '100%' }} className="form-control"
```

Comment:
`type` should be `url` See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url